scripts/backup

```diff
@@ -7,7 +7,7 @@
 #=================================================
 
 # source ../settings/scripts/_common.sh
-source /usr/share/yunohost/helpers
+source "/usr/share/yunohost/helpers"
```

Comment:
This is not an issue...
You DON'T need double quotes here !

Reply:
Just want to prevent anything looking like this :
https://github.com/MrMEEE/bumblebee-Old-and-abbandoned/commit/a047be85247755cdbe0acce6f1dafc8beb84f2ac#diff-3fbb47e318cd8802bd325e7da9aaabe8L351